generator/generator_config.textproto

```diff
@@ -47,6 +47,15 @@ service {
   retryable_status_codes: ["kUnavailable", "kUnknown"]
 }
 
+# API Keys
+service {
+  service_proto_path: "google/api/apikeys/v2/apikeys.proto"
+  product_path: "google/cloud/apikeys"
+  initial_copyright_year: "2022"
+  service_endpoint_env_var: "GOOGLE_CLOUD_CPP_APIKEYS_SERVICE_ENDPOINT"
```

Comment:
FYI: It would have generated `GOOGLE_CLOUD_CPP_API_KEYS_ENDPOINT`, which does not conflict with other names.

Reply:
Fixed. PTAL

Reply:
LGTM